packages/imperative/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+
+## Recent Changes
+
+- BugFix: Modified showMsgWhenDeprecated() to allow an empty string as a paramter when no replacement is availible for deprecated command to print an alternative message. [#2041](https://github.com/zowe/zowe-cli/issues/2041)
```

Comment:
- Please say what `showMsgWhenDeprecated()` is: Modified the `showMsgWhenDeprecated()` **command/property/cat breed** ...
- Fix typo in **paramter**
- This part of the sentence sounds a little awkward, not sure what you're saying: "for deprecated command to print an alternative message." Do you mean "for **_the_** deprecated command"?